src/ledger-iframe-bridge.ts

```diff
@@ -85,11 +86,21 @@ type IFramePostMessage<TAction extends IFrameMessageAction> =
     target: typeof LEDGER_IFRAME_ID;
   };
 
-export class LedgerIframeBridge implements LedgerBridge {
+const BRIDGE_URL = 'https://metamask.github.io/eth-ledger-bridge-keyring';
```

Comment:
the previous implementation was hardcoding this const URL at LedgerKeyring class, and pass in to LedgerIframeBridge

as the init function is called by metamask keyring, the init function will not pass in any parameter

here is the org code from ledgerKeyring
```
async init() {
    return this.bridge.init(this.bridgeUrl);
  }
```

the idea is not to init the bridgeUrl from ledgerKeyring, as it will make this high level interface only support iframe ledger bridge, instead we give the flexibility on the bridge class, to pass in any bridge url , if no bridge URL will be pass in , the const value will be applied
```
constructor(opts?: LedgerIframeBridgeOptions) {
    this.bridgeUrl = opts?.bridgeUrl ?? BRIDGE_URL;
  }
```

Reply:
Although keeping the old signature would be nice, I can see why we didn't do that here (that `bridgeUrl` might make no sense with some other bridge that doesn't use a webpage).

An alternative that comes to my mind would be keeping the `bridge options` in the `init` method instead of the `constructor`. But those would change type in any case so I'm not sure of the advantages of moving them back to `init`. I guess both works since it will be breaking in any case